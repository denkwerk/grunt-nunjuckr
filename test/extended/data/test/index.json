{
    "title" : "I am a template with a deep path!"
}