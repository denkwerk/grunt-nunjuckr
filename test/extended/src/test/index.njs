<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8">
    <title>Nunjuckr extended test</title>
</head>
<body>

    <h1>This is a extended test for nunjuckr.</h1>

    <h2><$ title $></h2>

</body>
</html>
