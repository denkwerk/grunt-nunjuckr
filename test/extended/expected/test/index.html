<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8">
    <title>Nunjuckr extended test</title>
</head>
<body>

    <h1>This is a extended test for nunjuckr.</h1>

    <h2>I am a template with a deep path!</h2>

</body>
</html>
